src/test/java/plugins/MavenPluginTest.java

```diff
@@ -246,4 +248,20 @@ private void assertHasModule(MavenModuleSet job, String name) {
         find(by.xpath("//a[@href='%s/']", name)).click();
         assertThat(driver.getCurrentUrl(), equalTo(build.module(name).url.toExternalForm()));
     }
+
+    @Issue("JENKINS-22252")
+    @WithPlugins({"maven-plugin@2.12", "tasks"})
```

Comment:
> there should be no version specified (presuming it does not depend on any recent features)

It does not depend on any recent features. But if you do not specify a version, then the test will *not* test the version on the update center, currently 2.12. Instead it will use the 2.7.1 or whatever bundled¹ with `jenkins.war`. This was the way I found to force it to install version 2.12 or newer so that the test is meaningful.

¹One of the many irritating consequences of plugin bundling—hope to see it die soon.